I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/day_4_branch_2.rpy", line 42, in script
    n "*[name] не успев и закричать о помощи, как был слышен выстрел из пистолета*"
  File "renpy/common/00nvl_mode.rpy", line 377, in do_display
    renpy.display_say(
OSError: Couldn't find file 'audio/pistol/mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/day_4_branch_2.rpy", line 42, in script
    n "*[name] не успев и закричать о помощи, как был слышен выстрел из пистолета*"
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 921, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\exports.py", line 1373, in say
    who(what, *args, **kwargs)
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\character.py", line 1266, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "renpy/common/00nvl_mode.rpy", line 377, in do_display
    renpy.display_say(
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\character.py", line 666, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\display\core.py", line 4103, in interact_core
    renpy.audio.audio.periodic()
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\audio\audio.py", line 1166, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\audio\audio.py", line 1144, in periodic_thread_main
    periodic_pass()
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\audio\audio.py", line 1086, in periodic_pass
    c.periodic()
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\audio\audio.py", line 517, in periodic
    topf = load(filename)
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\audio\audio.py", line 70, in load
    rv = renpy.loader.load(fn)
  File "C:\Users\dan03\Downloads\renpy-8.0.3-sdk\renpy\loader.py", line 820, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'audio/pistol/mp3'.

Windows-10-10.0.22000 AMD64
Ren'Py 8.0.3.22090602
Любовь, деньги, шизофрения 
Fri Sep  9 15:48:18 2022
